web/views/note_view.ex

```diff
@@ -1,4 +1,8 @@
 defmodule Notox.NoteView do
   use Notox.Web, :view
 
+  def render_markdown(markdown) do
+    import Earmark
+    Earmark.to_html(markdown) |> raw
```

Comment:
You don't actually want to strip out all HTML. HTML is valid in Markdown documents, and some people may want to include it in non-malicious ways. You just want to make sure that they can't include any JavaScript, which would allow them to do malicious things (we can talk more about this this week).

I would suggest rendering the markdown as HTML, then sanitizing it, then outputing the resulting HTML raw.